backend/src/main/java/ai/verta/modeldb/experimentRun/FutureExperimentRunDAO.java

```diff
@@ -44,6 +45,7 @@
   private final DatasetHandler datasetHandler;
   private final PredicatesHandler predicatesHandler;
   private final SortingHandler sortingHandler;
+  private final Config config = Config.getInstance();
```

Comment:
Get as constructor argument. Instantiating like this is bad practice as it doesn't allow us to change the config (e.g. testing).

Reply:
Removed and added feature flag in constructor